src/core_plugins/kibana/public/home/components/home.js

```diff
@@ -0,0 +1,117 @@
+          />
+        </KuiFlexItem>
+      );
+    });
+  };
+
+
+  return (
+    <div className="kuiView home">
+      <div className="kuiViewContent">
+
+        <div className="kuiViewContentItem kuiVerticalRhythmXLarge">
+          <KuiFlexGroup
+            className="kuiVerticalRhythmSmall"
+            justifyContent="spaceBetween"
+            alignItems="flexEnd"
+          >
+            <KuiFlexItem>
+              <h1 className="kuiTitle">
+                Welcome to Kibana
+              </h1>
+            </KuiFlexItem>
+
+            <KuiFlexItem grow={false}>
+              <KuiFlexGroup alignItems="center">
+                <KuiFlexItem grow={false}>
+                  <p className="kuiText kuiSubduedText">
+                    Data already in Elasticsearch?
+                  </p>
+                </KuiFlexItem>
+
+                <KuiFlexItem grow={false}>
+                  <KuiButton buttonType="secondary">
+                    <a href={addBasePath('/app/kibana#/management/kibana/index')}>Set up index patterns</a>
+                  </KuiButton>
+                </KuiFlexItem>
+              </KuiFlexGroup>
+
+            </KuiFlexItem>
+          </KuiFlexGroup>
+        </div>
+
+        <div className="kuiViewContentItem kuiVerticalRhythmXLarge">
+          <KuiFlexGroup className="kuiVerticalRhythm">
+            <KuiFlexItem className="kuiPanel">
+              <div className="kuiPanelBody">
+                <h3 className="kuiSubTitle kuiVerticalRhythm">
+                  Visualize and explore data
+                </h3>
+                <KuiFlexGrid className="kuiVerticalRhythmSmall topFeatures" columns={2}>
+                  { renderDirectories(directoryCategories.DATA) }
+                </KuiFlexGrid>
+              </div>
+            </KuiFlexItem>
+            <KuiFlexItem className="kuiPanel">
+              <div className="kuiPanelBody">
+                <h3 className="kuiSubTitle kuiVerticalRhythm">
+                  Manage and administer the Elastic stack
+                </h3>
+                <KuiFlexGrid className="kuiVerticalRhythmSmall topFeatures" columns={2}>
+                  { renderDirectories(directoryCategories.ADMIN) }
+                </KuiFlexGrid>
+              </div>
+            </KuiFlexItem>
+          </KuiFlexGroup>
+        </div>
+
+        <div className="kuiViewContentItem kuiVerticalRhythmXLarge center">
+
+          <h4 className="kuiSubduedText kuiVerticalRhythmSmall">
+            {`Didn't find what you were looking for?`}
+          </h4>
+
+          <KuiButton
+            className="kuiVerticalRhythmSmall"
+            buttonType="secondary"
+          >
+            <a href="#/home/feature_directory">View full directory of Kibana features</a>
+          </KuiButton>
+
+        </div>
+
+      </div>
+    </div>
+  );
+}
+
+Home.propTypes = {
+  addBasePath: PropTypes.func.isRequired,
+  directories: PropTypes.object.isRequired,
```

Comment:
I think what might help decouple this from our angular registry format is to pass `directories.inTitleOrder` into all these components, and have them just take an array instead of an object.

e.g. down in index.js change
`$scope.directories = Private(FeatureCatalogueRegistryProvider);` 
to 
`$scope.directories = Private(FeatureCatalogueRegistryProvider).inTitleOrder;`

What do you think?  Then it'd be easier to express the shape better and you could change PropTypes.object to

```
PropTypes.arrayOf(PropTypes.shape({
  id: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  icon...
  showOnHomePage...
  category.....
  })),
```